src/commands/userinfo_command.py

```diff
@@ -0,0 +1,10 @@
+import discord
+
+###TODO Command struggles with special characters, figure it out
+async def userinfo_command(ctx, self, user, guild):
+    member:str = discord.utils.find(lambda m: m.name == user, guild.members)
```

Comment:
respect for using a lambda here

Reply:
Don't think that part works yet, hold the respect a little longer lol